modules/apps/static/portal-configuration/portal-configuration-persistence-impl/src/main/java/com/liferay/portal/configuration/persistence/internal/ConfigurationPersistenceManager.java

```diff
@@ -410,14 +407,10 @@ private void _deleteFromDatabase(String pid) throws IOException {
 	}
 
 	private File _getConfigFile(String fileName) throws IOException {
-		Path configsDirPath = Paths.get(
-			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR);
-
-		configsDirPath = configsDirPath.toRealPath();
-
-		Path configFilePath = configsDirPath.resolve(fileName);
+		File configFile = new File(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR, fileName);
 
-		return configFilePath.toFile();
+		return configFile.getAbsoluteFile();
```

Comment:
Hi @yuhai ,

Just wanted to make sure, would this change re-introduce the issue in [LPS-113749](https://issues.liferay.com/browse/LPS-113749)?

The full test would be to combine steps from both [LPS-113749](https://issues.liferay.com/browse/LPS-113749) and [LPS-162706](https://issues.liferay.com/browse/LPS-162706)

Reply:
**Additional Info - Debugging**
For my testing, I found it to be important to test during startup. Here are two breakpoints I used to track how felix fileinstall filename is stored:
- [ConfigurationFileInstaller.java#L141-L149](https://github.com/liferay/liferay-portal/blob/7.4.3.42-ga42/modules/apps/static/portal-file-install/portal-file-install-impl/src/main/java/com/liferay/portal/file/install/internal/configuration/ConfigurationFileInstaller.java#L141-L149)
- [ConfigurationPersistenceManager.java#L614-L620](https://github.com/liferay/liferay-portal/blob/7.4.3.42-ga42/modules/apps/static/portal-configuration/portal-configuration-persistence-impl/src/main/java/com/liferay/portal/configuration/persistence/internal/ConfigurationPersistenceManager.java#L614-L620)

Reply:
Hi @ericyanLr,

Thanks for pointing out the issue. The previous fix didn't work for [LPS-113749](https://issues.liferay.com/browse/LPS-113749).  I just add the new last commit to fix the case. Please see https://github.com/liferay-core-infra/liferay-portal/pull/1772/commits/99edf76f10dacd337ea947c4cfc0e6c8ad6d124c